components/server/src/api/organization-service-api.ts

```diff
@@ -310,6 +313,33 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
             }
         }
 
+        if (!req.updateMaxParallelRunningWorkspaces && typeof req.maxParallelRunningWorkspaces === "number") {
+            throw new ApplicationError(
+                ErrorCodes.BAD_REQUEST,
+                "updateMaxParallelRunningWorkspaces is required to be true when updating maxParallelRunningWorkspaces",
```

Comment:
This sounds way to technical for an API-level error.
What are we after here, in user terms? What is `updateMaxParallelRunningWorkspaces` actually about? :thinking:

Reply:
It actually does not look like we need it here 🤔. I thought we did at first, but since our settings updates are never truly partial (and contrary to what I originally believed, gRPC does not seem to confuse `0` and `undefined`), so let's get rid of it.